src/hooks/usePageViews.js

```diff
@@ -59,7 +73,7 @@ const ANALYTICS_PROVIDERS = {
  * We assume that the provider has been included already in the global scope of the
  * containing page where the SDK is embedded.
  */
-const trackPageView = location => {
+const trackPageView = (location, previousLocation) => {
   const promises = Object.values(ANALYTICS_PROVIDERS).map(callback => callback(location));
```

Comment:
```suggestion
  const promises = Object.values(ANALYTICS_PROVIDERS).map(callback => callback(location, previousLocation));
```
you still need to actually pass the argument to the analytics-specific function too